laurent22/joplin
# Desktop: Update bundled Backup plugin to v1.4.2

# Summary

Updates the bundled Backup plugin from v1.4.1 to v1.4.2.

See https://github.com/JackGruber/joplin-plugin-backup/compare/52d898315cab259da638698cc41120e9fae593ef..5ba57c18ac0f24f20832f012e015a080b138f0c4

# Testing plan


Testing plan (Ubuntu 24.04):
1. Click Tools > Create backup.
2. Verify that a "backup completed" dialog is visible.
3. Enable password protected backups
4. Repeat steps 1-2.

Also see the [Backup integration tests](https://github.com/laurent22/joplin/blob/dev/packages/app-desktop/integration-tests/simpleBackup.spec.ts).
